pyodide_build/pywasmcross.py

```diff
@@ -105,7 +105,7 @@ def capture_compile(args):
         sys.exit(result.returncode)
 
 
-def handle_command(line, args):
+def handle_command(line, args, pretend=False):
```

Comment:
I usually see this called `dryrun`.  And maybe prefix it with `_` so that it's clear it's a testing detail?